api-admin/src/main/java/bio/terra/pearl/api/admin/service/notifications/NotificationConfigExtService.java

```diff
@@ -133,6 +139,46 @@ public NotificationConfig create(
     return create(newConfig, studyEnvironment, portalEnvironment);
   }
 
+  /**
+   * Deletes the config specified by id, ensuring it belongs the appropriate study and environment
+   */
+  @Transactional
+  public void delete(
+      AdminUser user,
```

Comment:
we've recently tried to start using "operator" instead of "user" for this argument name to clarify that this is the adminUser making the request, as opposed to a target of the method

Reply:
That makes a lot of sense, I went ahead and refactored the rest of the file as it had `user` for everything.